compat/os/mingw.h

```diff
@@ -46,7 +46,9 @@ fsync(int fd)
 }
 #endif
 
-#define mkdir(p,F) mkdir((p))
+/* importaed from mswindows.h */
```

Comment:
1) This info only matters if the two need to be kept in sync. Implementation in `mswindows.h` is for a different compiler and may thus expected to be different when that compiler build gets fixed.

2)  If there is a copyright assigned to this implementation. the legal requirement t is to duplicate the relevant copyright blurb and update CREDITS.
  - since you, me and Guido (AFAIK the relevant copyright holders for this code) have all delegated GPLv2, this case is not applicable.

```suggestion
```

That said, I do hope you can replace these `#define` with `inline const` whenever possible.